providers/common/include/prov/hybrid_pkey.h

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2024 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include <openssl/evp.h>
+#include "internal/refcount.h"
+
+#define MAX_HYBRID_ALGS 2
+#define MAX_HYBRID_PARAMS 20
+
+typedef struct {
+    const unsigned int num_algs;
+    const struct {
+        const char *name;
+        size_t pubkey_length_bytes;
```

Comment:
This looks like it is geared towards KEM.. Do you imagine this being used for signatures? Is that a separate structure for that case?
Should this be called HybridKEM?

Reply:
If/when other hybrid operations are defined, I anticipate that this `info` structure will be extended to suite.

It's storing constant information about the key.  This should include everything that is relevant for any operation the key supports.  However, until that information is actually required, I see no reason including it.

Using separate structure would require separate key management since this is primarily used there.